packages/camera/camera_android_camerax/README.md

```diff
@@ -60,6 +60,19 @@ requires specifying the `WRITE_EXTERNAL_STORAGE` permission (see [the CameraX do
 This is already done in the plugin, so no further action is required on your end. To understand
 the implications of specificying this permission, see [the `WRITE_EXTERNAL_STORAGE` documentation][11].
 
+Please note that the [`READ_EXTERNAL_STORAGE`][13] permission may be implied from `WRITE_EXTERNAL_STORAGE`
+permission and thus, be included in the merged Android manifest of your app. If you do not want the
+`READ_EXTERNAL_STORAGE` permission to be included in the merged Android manifest of your app, then you can
+do the following to remove it:
+
+1. Ensure that your app nor any of the plugins that it depends on require the `READ_EXTERNAL_STORAGE` permission.
+2. Add the following to your app's `your_app/android/app/src/main/AndroidManifest.xml`:
+
+```xml
+  <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"
+    tools:node="remove" />
+```
+
```

Comment:
Please link to the documentation for camerax and the reasons for why this permission is included.

Reply:
https://developer.android.com/media/camera/camerax/architecture#permissions

Reply:
This is linked above this part:

```md
In order to save captured images and videos to files on Android 10 and below, CameraX
requires specifying the `WRITE_EXTERNAL_STORAGE` permission (see [the CameraX documentation][10]).
This is already done in the plugin, so no further action is required on your end. To understand
the implications of specificying this permission, see [the `WRITE_EXTERNAL_STORAGE` documentation][11].
```

Is that sufficient?

Reply:
"To understand the implications of specificying(sp?) this permission, see [the `WRITE_EXTERNAL_STORAGE` documentation][11]." 

Is technically correct but not as useful. Developers like the original reporter of https://github.com/flutter/flutter/issues/131116 are being asked by their legal team to fill out a justification for a permission. One they didnt add do not actually call. 

I think it should be changed to. "If you want to understand the privacy impact of `WRITE_EXTERNAL_STORAGE` see see [the `WRITE_EXTERNAL_STORAGE` documentation][11]. We have seen apps also have `READ_EXTERNAL_STORAGE` permission automatically added this appears to be implied from `WRITE_EXTERNAL_STORAGE`."